### ISSUE TITLE
how to set service port name

### ISSUE THREAD

**Issue by jiuchongxiao** [rancher/rke#817]
Rancher versions:
2.0.6

when i new ingress in load balancing for workloads,it will product a ingress and a service discovery.
the service discovery's yaml will as below:

  ports:
  - nodePort: 32399
    port: 9102
    protocol: TCP
    targetPort: 9102
but i expected it be:

ports:
 - nodePort: 32399
   name: http
   port: 9102
   protocol: TCP
   targetPort: 9102
i can sovle it by copy the service yaml file to xxx.yaml and increase the port name,then kubectl apply -f xxx.yaml.
but i want if is it has a method resolve it by rke config file.

https://github.com/rancher/rancher/issues/14814

**Comment by galal-hussein** [comment]
@jiuchongxiao i don't think this is rke issue, Rancher platform creates the the ingress not rke
